packages/core/demo/demo-data/line.ts

```diff
@@ -130,6 +130,9 @@ export const lineOptions = {
 			]
 		}
 	},
+	points: {
```

Comment:
We shouldn't replace our demo with this. We can have a separate demo if needed.

Reply:
Why wouldn't we? We allow custom radius of the points now, how else would someone find out?

Reply:
Not that we shouldn't demo the functionality, we shouldn't _replace_ defaults with a custom value without making it obvious it's a customization.
Having an interactive circle size demo or a separate demo for the functionality.
Documentation.